src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -63,6 +64,13 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
                 }
             }
 
+            if ($question instanceof ConfirmationQuestion) {
+                $value = trim(fgets(STDIN));
+                if ($value === 'y') {
```

Comment:
Any true-ish value should be considered according to https://github.com/symfony/symfony/blob/c1bca297cf15145fff29ee169e9820ffacc5d37d/src/Symfony/Component/Console/Question/ConfirmationQuestion.php#L32
I think it should read more bytes from the input stream (as much as in `doAsk()`) and, if it's not `false`, return the value after it has been passed to `Question::getNormalizer()/I getValidator()`.

Also, couldn't this work for any question instead of only confirmation questions?